### PR TITLE
Hotfix Filetree #333

### PR DESCRIPTION
**Pull request by zonky2**
set template to protected and implement getter and setter

### CODE REVIEW

File: src/ContaoCommunityAlliance/DcGeneral/Contao/View/Contao2BackendView/Widget/FileTree.php [hunk 1]
```diff
@@ -44,7 +44,7 @@ class FileTree extends AbstractWidget
      *
      * @var string
      */
-    private $subTemplate = 'widget_filetree';
+    protected $subTemplate = 'widget_filetree';
```
Comment (discordier) [inline src/ContaoCommunityAlliance/DcGeneral/Contao/View/Contao2BackendView/Widget/FileTree.php]: Why did you change this?
IMO this is wrong, as the value can be modified via getter and setter and no one should manipulate it directly.
Can you explain to me your reasoning?

Comment (zonky2) [inline src/ContaoCommunityAlliance/DcGeneral/Contao/View/Contao2BackendView/Widget/FileTree.php]: look at https://github.com/contao-community-alliance/dc-general/issues/333 ...
I have uploadet files from a frontend form and show the files at the backend - at a MM input mask... this works fine, but the editor should open the file (a PDF) via link in a new tab... and I override the template with my own

Comment (discordier) [inline src/ContaoCommunityAlliance/DcGeneral/Contao/View/Contao2BackendView/Widget/FileTree.php]: But change to `protected` is not needed at all!

Comment (zonky2) [inline src/ContaoCommunityAlliance/DcGeneral/Contao/View/Contao2BackendView/Widget/FileTree.php]: you are the master... if you sure, I`ll change it...

**Comment by discordier** [comment]
Merged in 88192fadca80d0603b08b4f920ba2b0a1472d3b4
